File: sample-chat-java/app/src/main/java/com/quickblox/sample/chat/java/utils/imagepick/OnImagePickedListener.java
package com.quickblox.sample.chat.java.utils.imagepick;

import java.io.File;

public interface OnImagePickedListener {

    void onImagePicked(int requestCode, File file);

    void onImagePickError(int requestCode, Exception e);

    void onImagePickClosed(int requestCode);
}

File: sample-chat-kotlin/app/src/main/java/com/quickblox/sample/chat/kotlin/utils/imagepick/OnImagePickedListener.kt
package com.quickblox.sample.chat.kotlin.utils.imagepick

import java.io.File


interface OnImagePickedListener {

    fun onImagePicked(requestCode: Int, file: File)

    fun onImagePickError(requestCode: Int, e: Exception)

    fun onImagePickClosed(requestCode: Int)
}